roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -114,6 +114,12 @@ filestore split multiple = {{ filestore_split_multiple }}
 osd op threads = {{ osd_op_threads }}
 filestore op threads = {{ filestore_op_threads }}
 filestore max sync interval = {{ filestore_max_sync_interval }}
+{% if filestore_xattr_use_omap is not None %}
```

Comment:
Does this need to be `{% if filestore_xattr_use_omap == none %}`?  I'm wondering if this is what caused the run failure.